planemo/commands/cmd_shed_create.py

```diff
@@ -14,6 +14,13 @@
 @options.shed_message_option()
 @options.shed_skip_upload()
 @command_function
+@click.option(
+    "--noverify",
+    is_flag=True,
+    help="When set, Requests will trust SSL certificates for HTTPS requests. "
+         "It sets verify=False in requests call",
+    default=False,
+)
```

Comment:
Since this is reused multiple times, it should be moved from the `cmd_` files to `planemo/options.py` as:

```python
def noverify_option():
    return planemo_option(
        "--noverify",
        is_flag=True,
        default=False,
        help="When set, Requests will trust SSL certificates for HTTPS requests. "
             "It sets verify=False in requests call",
    )
```

and then add `noverify_option()` to `shed_target_options()` and `engine_options()` in the same file.

This will add the `--noverify` option also to the `shed_serve` and `run` commands.

Reply:
ok, thank you. I've done the changes.